lib/src/isolate_manager/isolate_manager_web.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:async';
+import 'dart:developer';
+
+import 'package:dash_kit_network/dash_kit_network.dart';
+import 'package:dash_kit_network/src/isolate_manager/isolate_manager.dart';
+
+// ignore: prefer-match-file-name
+class IsolateManager extends IsolateManagerInterface {
+  IsolateStatus _isolateStatus = IsolateStatus.created;
+
+  @override
+  Future<void> start() async {
+    assert(_isolateStatus == IsolateStatus.created, 'Isolate already started');
+    _isolateStatus = IsolateStatus.initialized;
+
+    return;
+  }
+
+  @override
+  void stop() {
+    _isolateStatus = IsolateStatus.stopped;
+
+    return;
+  }
+
+  @override
+  Future<FutureOr<T>> send<T>(
+    Response<dynamic> response,
+    FutureOr<T> Function(Response<dynamic>) mapper,
+  ) async {
+    try {
+      return await mapper(response);
+    } catch (e, stackTrace) {
```

Comment:
Do we really need try/catch here? We already on main isolate and should throw any exception anyway.